.travis.yml

```diff
@@ -45,6 +164,8 @@ notifications:
   webhooks:
     urls:
       - https://webhooks.gitter.im/e/b0312b18473340459d3e
+    # NOTE: Be silent about CI, until some PR started to pass CI succesfully.
     on_success: change
-    on_failure: always
+    # NOTE: `master` status in on the front page badge.
+    on_failure: never
     on_start: never
```

Comment:
Is this for https://gitter.im/haskell-nix/Lobby?

I don't see any notifications there (and I personally don't need any).

Reply:
Yes.

"I personally don't need any" so (`on_failure: always -> never`) at the least.

I also think that is a pretty unneeded feature, nobody would run into PRs through the CI notification, since that notification does not mean the work is indeed ready.